source/MissionAction.cpp

```diff
@@ -202,7 +215,9 @@ void MissionAction::Load(const DataNode &node, const string &missionName)
 		{
 			int count = (child.Size() < 3 ? 1 : static_cast<int>(child.Value(2)));
 			if(count >= 0)
+            {
 				requiredOutfits[GameData::Outfits().Get(child.Token(1))] = count;
+            }
```

Comment:
Style: unnecessary braces.
```suggestion
				requiredOutfits[GameData::Outfits().Get(child.Token(1))] = count;
```